platforms/documentation/docs/src/docs/userguide/redirects/resolution_stategy_tuning.adoc

```diff
@@ -0,0 +1,3 @@
+++++
+<meta http-equiv="refresh" content="0;URL=resolution_strategy_tuning.html">
```

Comment:
💅 `resolution_strategy_tuning.html` is itself a redirect to `dependency_resolution_basics.html#dependency-resolution-basics`

Reply:
I put this file back in this PR: https://github.com/gradle/gradle/pull/31672/files#diff-6f89b400fe27d44c5328128ffb4aece829047885305696c01ab81c72b28af7e0 so I will remove this.

Reply:
Looks like this is a complete mistake. Removed.